asapdiscovery-docking/asapdiscovery/docking/docking_v2.py

```diff
@@ -156,6 +196,43 @@ def write_docking_files(
             combined_oemol = result.to_posed_oemol()
             save_openeye_pdb(combined_oemol, output_pdb_file)
 
+            results_df = result.make_df_from_docking_results([result])
+            results_df.to_csv(compound_dir / "docking_results.csv")
+
+    def check_results_exist(
+        self, input: DockingInputPair, output_dir: Union[str, Path]
+    ) -> bool:
+        """Using the same logic as write_docking_files, check if results exist"""
+        output_pref = (
+            input.complex.target.target_name + "_+_" + input.ligand.compound_name
+        )
+        compound_dir = Path(output_dir) / output_pref
+
+        sdf_file = compound_dir / "docked.sdf"
+        pdb_file = compound_dir / "docked_complex.pdb"
+
+        return sdf_file.exists() and pdb_file.exists()
+
+    def get_unfinished_results(
```

Comment:
We can embed this inside the docking function, I don't think it needs to be seperately dispatched to dask.